packages/openapi-v3-types/src/openapi-v3-spec-types.ts

```diff
@@ -0,0 +1,395 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/openapi-v3-types
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+/*
+ * OpenApiSpec - A typescript representation of OpenApi 3.0.0
+ */
+
+// tslint:disable:max-line-length
+
+import * as OAS3 from 'openapi3-ts';
+// Export spec interfaces from the community module if missing in our package
+export * from 'openapi3-ts';
+
+export type OpenApiSpec = OAS3.OpenAPIObject;
+/**
+ * Custom extensions can use arbitrary type as the value,
+ * e.g. a string, an object or an array.
+ */
+// tslint:disable-next-line:no-any
+export type ExtensionValue = any;
+
+/**
+ * The location of a parameter.
+ * Possible values are "query", "header", "path" or "body".
+ * <p>Specification:
```

Comment:
Please remove the HTML tags (`<p>`)

Reply:
@kjdelisle inherited those tags from v2: https://github.com/strongloop/loopback-next/blob/master/packages/openapi-spec/src/openapi-spec-v2.ts

Just curious.... any harm from it?

Reply:
@kjdelisle ^^^ could you elaborate more about your comment? Thanks!

Reply:
talked with Kevin, I thought jsdoc has its own syntax/parsing convention, but that's not true, any arbitrary html tag only mess up the jsdoc parsing. Will remove them then.